core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -126,8 +127,7 @@ public void initialize(String name, Map<String, String> unresolved) {
     // catalog service
     Map<String, String> props = EnvironmentUtil.resolveAll(unresolved);
 
-    long startTimeMillis =
-        System.currentTimeMillis(); // keep track of the init start time for token refresh
+    Instant startTime = Instant.now(); // keep track of the init start time for token refresh
```

Comment:
Throughout Iceberg, we use `System.currentTimeMillis()`. Is there a reason to deviate from that?

Reply:
calculations with an `Instant` are generally easier. Additionally we don't have to add `xyzMillis` to every variable to indicate that those are milliseconds